mscore/inspector/voicing_select.ui

```diff
@@ -118,12 +115,12 @@
      </item>
      <item>
       <property name="text">
-       <string>Stop at measure</string>
+       <string>Stop at measure end</string>
       </property>
      </item>
      <item>
       <property name="text">
-       <string>Segment duration</string>
+       <string>Chord/rest duration</string>
```

Comment:
This is what it is doing, in 'layperson' terms

Reply:
Shouldn't these all be title case?

Reply:
I though about that but then was unsure about the rules there